ros2/pytest_wrapper.py.tpl

```diff
@@ -41,28 +39,22 @@ def main() -> None:
     if 'ROS_LOG_DIR' not in os.environ:
         os.environ['ROS_LOG_DIR'] = bazel_test_output_dir
 
-    with contextlib.ExitStack() as stack:
```

Comment:
I would keep this as an extra line of defense. I also find it good practice to use tempfile and not rely on Bazel for stuff like this. But if you really like it, we can always patch locally.